src/components/inputs/money-input/README.md

```diff
@@ -32,23 +32,22 @@ The amount can have an arbitrary precision. When the precision of the amount exc
-| `value`                | `{ currencyCode: string, amount: string }`                      |    ✅    | -                            | -       | Value of the input. Consists of the currency code and an amount. `amount` is a string representing the amount. A dot has to be used as the decimal separator. |
-| `value.currencyCode`   | `string`                                                        |    ✅    | -                            | -       | The currency code influences the fraction digits when formatting on blur. Can be an empty string.                                                             |
-| `value.amount`         | `string`                                                        |    ✅    | -                            | -       | Amount formatted as a number, e.g. "10.30". Must use a dot as the decimal separator. Can be an empty string.                                                  |
-| `currencies`           | array of `string`                                               |    -     | -                            | []      | List of possible currencies. When not provided or empty, the component renders a label with the value's currency instead of a dropdown.                       |
-| `placeholder`          | `string`                                                        |    -     |                              | -       | Placeholder text for the input.                                                                                                                               |
-| `isDisabled`           | `bool`                                                          |    -     | -                            | `false` | Indicates that the field cannot be used.                                                                                                                      |
-| `onBlur`               | `func`                                                          |    -     | -                            | -       | Called when the amount field or the currency code dropdown is blurred.                                                                                        |
-| `onChange`             | `function(event)`                                               |    ✳️    | -                            | -       | Called with the event of the input or dropdown when either the currency or the amount have changed. Either `onChange` or `onChangeValue` must be passed.      |
-| `onChangeValue`        | `function(nextValue: { currencyCode: string, amount: string })` |    ✳️    | -                            | -       | Called with the full value when either the currency or the amount have changed. Either `onChange` or `onChangeValue` must be passed.                          |
```

Comment:
`onChangeValue` This does no longer exist and was removed.  `onChange` was adapted as well.